### PR TITLE
DO NOT MERGE UNTIL JUNE 4TH

### PR DESCRIPTION
**Pull request by vmbrasseur**
Draft of CFP announcement/instructions.

_None_ of this is approved by the programme committee yet.

### CODE REVIEW

**Comment by vmbrasseur** [comment]
Suggestion: don't use the word "blind".

**Comment by vmbrasseur** [comment]
Suggestion: Say "Seattle area" rather than just Seattle.

File: _posts/2018-06-04-CFP-open.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+title: 'The 2018 SeaGL CFP is open for business!'
+status: publish
+type: post
+published: true
+categories: news
+---
+
+_Calling all speakers or speakers-to-be! Our 2018 Call for Proposals is open!_
+
+SeaGL is dedicated to supporting newer local Seattle speakers. We welcome speakers of **all backgrounds and levels of experience** – even if you’ve never spoken at a technical conference. If you’re excited about Free/Libre/Open technologies or communities, then we want to hear from you!
```
[the remark this answers is not in the document]
Comment (genehack) [reply _posts/2018-06-04-CFP-open.md]: I see in the meeting notes this was maybe going to be refined to "Seattle area"; I'd suggest going slightly broader and making it more PNW targeted. #2¢

[the remark this answers is not in the document]
Comment (nhandler) [reply _posts/2018-06-04-CFP-open.md]: I saw that. I have no objections to broadening Seattle to PNW, but my comment was referring to adding something like, `(but we are open and welcoming to speakers from all locations)`.

File: _posts/2018-06-04-CFP-open.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+[Click here to propose a talk.](https://osem.seagl.org/conferences/seagl2018/program/proposals)
+
+## CFP dates
+
+* CFP Opens: June 4th, 2018
+* CFP Closes: July 29th, 2018 - Midnight PDT
+* Speaker Notifications: September 3rd, 2018
```
[the remark this answers is not in the document]
Comment (vmbrasseur) [reply _posts/2018-06-04-CFP-open.md]: Like last year, we'll be notifying everyone by/on this date.

(or that's the plan)

File: _posts/2018-06-04-CFP-open.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+* Schedule Published: September 17th, 2018
+
+## Talk formats
+
+Like last year, in 2018 SeaGL is looking for talks in two formats:
+
+* **20 minutes**: Introduce the audience to a new technology, concept, or just recap an older idea which you think is really neat.
+* **50 minutes**: Go more in depth! Do a demo! This is your chance to really educate the audience about something you enjoy.
+
+Both of these time slots **include the Q&A time**. Please time your presentations accordingly. We suggest aiming for a 15 minute presentation for the 20 minute time slot and for 40-45 minutes for the 50 minute time slot, but as the speaker the final presentation time is up to you (as long as you don't exceed your time slot).
```
[the remark this answers is not in the document]
Comment (vmbrasseur) [reply _posts/2018-06-04-CFP-open.md]: Yeah nah. Comma overload.

File: _posts/2018-06-04-CFP-open.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+## Proposal and public speaking resources
+
+Never presented at a conference or meetup before? Presented but still not feeling confident? It's OK, even the most experienced conference presenters aren't necessarily confident at this stuff.
+
+The [Public_Speaking](https://github.com/vmbrasseur/Public_Speaking) repository has collected a lot of resources to help you level up in your conference presenting.
+
+Pay particular attention to the [Proposing talks](https://github.com/vmbrasseur/Public_Speaking#proposing-talks) section of this page. Follow these tips and your talk proposals will stand a better chance of being selected.
+
+**PLEASE NOTE**: Whether your talk is accepted or not often doesn't have as much to do with how great your proposal is as it does with how many speaking slots the conference has available and the balance of the program the organisers need to craft. It's nothing personal: we just don't have enough time to accept all the great talk proposals we receive.
```
[the remark this answers is not in the document]
Comment (vmbrasseur) [reply _posts/2018-06-04-CFP-open.md]: There's already a plan for that. People whose talks are rejected from SeaGL will receive an email including a "but your talk may be a great fit for an opensource.com article…" blurb.

File: _posts/2018-06-04-CFP-open.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+## Help and mentoring for your proposals
+
+Want to propose a talk but want feedback on your idea, proposal wording, talk title or just on how to deal with nerves? The speaker committee is running CFP office hours during the CFP. We'll do everything possible to help you be successful with your proposal and presenatation. Office hour times: 
+
+* 12-1 PM Pacific Time, every Thursday in July until July 29th
+
+All office hours are held in the `#seagl` IRC channel on Freenode IRC. Don't worry if you're not familiar with IRC. Just [click here for the webchat](https://webchat.freenode.net/?channels=%23seagl), choose a nickname, and you're good to go!
+
+If you'd like assistance outside of the office hours, please email us at [cfp-help@seagl.org](mailto:cfp-help@seagl.org). We welcome all questions that arrive between June 4th and July 29th.
```
[the remark this answers is not in the document]
Comment (vmbrasseur) [reply _posts/2018-06-04-CFP-open.md]: That's covered on the rest of the website.

File: _posts/2018-06-04-CFP-open.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+## Speaker travel support
```
[the remark this answers is not in the document]
Comment (vmbrasseur) [reply _posts/2018-06-04-CFP-open.md]: That will be covered elsewhere on the website (if it's covered at all; IIRC we rarely supply this information as the assumption is most attendees are local).

**Comment by vmbrasseur** [comment]
OK, I think this is good to go now. I'll merge tomorrow morning.

**Pull request merged**